### DISCUSSION TITLE
Support for vue 2

### DISCUSSION THREAD

**Discussion by joaopedrodcf** [unovue/reka-ui#282] [General]
Hey guys,

Just learned about this amazing library and wanted to know if it supports vue 2 or just vue 3.

Thanks

**Comment by danulqua** [comment]
@joaopedrodcf Hello!

Radix-Vue is built using Vue 3, so it definitely isn't meant to be used with Vue 2.

*If it answers your question, make sure to mark it as the answer.*

**Reply by zernonia** [reply]
Yeah it is built using Vue3. However thanks to Vue-demi, I believe we can visit this again in the future 😁

**Reply by joaopedrodcf** [reply]
Thanks gonna mark it as solved 👌
